code/modules/mining/mine_turfs.dm

```diff
@@ -135,6 +135,12 @@
 	spread = 1
 	scan_state = "rock_Silver"
 
+/turf/closed/mineral/titanium
+	mineralType = /obj/item/weapon/ore/titanium
+	spreadChance = 5
+	spread = 1
+	scan_state = "rock_Titanium"
```

Comment:
This icon is missing the fade-out animation in the file; it exists but it doesn't fade out like the rest do